### PR TITLE
adds bmlt workflow

### PR DESCRIPTION
**Pull request by tempsaint**
This needs to be pulled down and tested before merge (no local dev env)

### CODE REVIEW

File: bmlt-versions.php [hunk 1]
```diff
@@ -276,6 +286,43 @@ public function bmltVersionsSimpleFunc($atts = [])
...
+                    $workflow_content .= '<li class="bmlt_versions_li">';
+                    $workflow_content .= '<a href="'. $workflowDocs .'">View Documentation</a>';
+                    $workflow_content .= '</li>';
+                }
+                $workflow_content .= '<li class="bmlt_versions_li">';
+                $workflow_content .= '<a href="https://github.com/bmlt-enabled/bmlt-workflow" target="_blank">View On Github</a>';
+                $workflow_content .= '</li>';
+
+                $workflow_content .= '<li class="bmlt_versions_li">';
+                $workflow_content .= '<a href ="https://github.com/bmlt-enabled/bmlt-workflow/releases/download/' . $workflow_version . '/workflow-' . $workflow_version . '.zip" id="bmlt_versions_release">Download Latest Release</a>';
```
Comment (pjaudiomv) [inline bmlt-versions.php]: for this plugin the zip releases arent versioned, they are always `bmlt-workflow.zip`. 
so `/workflow-' . $workflow_version . '.zip` can just be `/bmlt-workflow.zip`

File: bmlt-versions.php [hunk 2]
```diff
@@ -577,6 +626,27 @@ public function bmltVersionsFunc($atts = [])
...
+                $workflow_content = '<div class="bmlt_versions_div github">';
+                $workflow_content .= '<ul class="bmlt_versions_ul">';
+                $workflow_content .= '<li class="bmlt_versions_li" id="bmlt-versions-workflow">';
+                $workflow_response = $this->githubLatestReleaseInfo('bmlt-workflow');
+                $workflow_version = $this->githubLatestReleaseVersion($workflow_response);
+                $workflow_date = $this->githubLatestReleaseDate($workflow_response);
+                $workflow_date_ver = $workflow_version . ' (' . date("m-d-Y", strtotime($workflow_date)) . ')';
+                $workflow_content .= '<strong>BMLT Workflow</strong><br>';
+                $workflow_content .= $this->githubReleaseDescription('workflow') . '<br><br>';
+                $workflow_content .= 'Latest Release : <strong><a href ="https://github.com/bmlt-enabled/bmlt-workflow/releases/download/' . $workflow_version . '/workflow-' . $workflow_version . '.zip' . '" id="bmlt_versions_release">' . $workflow_date_ver. '</a></strong>';
```
Comment (pjaudiomv) [inline bmlt-versions.php]: for this plugin the zip releases arent versioned, they are always `bmlt-workflow.zip`. 
so `/workflow-' . $workflow_version . '.zip` can just be `/bmlt-workflow.zip`

File: bmlt-versions.php [hunk 3]
```diff
@@ -577,6 +626,27 @@ public function bmltVersionsFunc($atts = [])
...
+            if ($workflow) {
+                $workflow_content = '<div class="bmlt_versions_div github">';
+                $workflow_content .= '<ul class="bmlt_versions_ul">';
+                $workflow_content .= '<li class="bmlt_versions_li" id="bmlt-versions-workflow">';
+                $workflow_response = $this->githubLatestReleaseInfo('bmlt-workflow');
+                $workflow_version = $this->githubLatestReleaseVersion($workflow_response);
+                $workflow_date = $this->githubLatestReleaseDate($workflow_response);
+                $workflow_date_ver = $workflow_version . ' (' . date("m-d-Y", strtotime($workflow_date)) . ')';
+                $workflow_content .= '<strong>BMLT Workflow</strong><br>';
+                $workflow_content .= $this->githubReleaseDescription('workflow') . '<br><br>';
```
Comment (pjaudiomv) [inline bmlt-versions.php]: release description should be repo name `bmlt-workflow` instead of just `workflow`

**Review by pjaudiomv — COMMENTED** [review]
looks pretty good thanks. couple comments but i think thats about it

**Comment by tempsaint** [comment]
Thanks @pjaudiomv - something is still broken with it. Not sure where to find the error. I try running activating the plugin in a docker instance and it crashing the site.

**Comment by GarrettWeinberg** [comment]
tested and confirmed working

**Pull request merged**